### PR TITLE
Code review changes to address Richard's comments.

### PR DESCRIPTION
**Pull request by sumitopensource**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/hughsie/fwupd/wiki/New-plugin-checklist))
- [] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,3 +125,5 @@ There are several automated fuzzing tests in fwupd. These take some time to run:
     ninja fuzz-synaptics-rmi
     ninja fuzz-firmware
     ninja fuzz-smbios
+
+
```
Comment (superm1) [inline README.md]: why did you change this?

Comment (sumitopensource) [reply README.md]: Change in number of Hypens below "Other frontends".
This change had gone by mistake in last merge. And Richard had reverted this in his commit. I did accept this change from Richard.

**Comment by hughsie** [comment]
Lets just use  sumitopensource:wip/sumit/cypress for now.
